src/index.d.ts

```diff
@@ -327,12 +327,12 @@ export {
   VariablesAreInputTypesRule,
   VariablesInAllowedPositionRule,
   // SDL-specific validation rules
-  LoneSchemaDefinitionRule,
-  UniqueOperationTypesRule,
-  UniqueTypeNamesRule,
-  UniqueEnumValueNamesRule,
+  LoneSchemaDefinitionRuleRule,
+  UniqueOperationTypesRuleRule,
+  UniqueTypeNamesRuleRule,
+  UniqueEnumValueNamesRuleRule,
```

Comment:
@IvanGoncharov I'm worried there may have been a mistake via find and replace here and in other places, can you take another look please?

Reply:
@trevor-scheer Thanks a lot for reporting 👍 
Fixed it here: https://github.com/graphql/graphql-js/pull/2407